File: apps/site/snippets/perf/low-power.kt
package snippets.perf

import androidx.compose.foundation.layout.size
import androidx.compose.runtime.Composable
import androidx.compose.ui.Modifier
import androidx.compose.ui.unit.dp
import dev.sinua.view.FxLowPower
import dev.sinua.view.generated.SinuaOrb
import dev.sinua.view.generated.SinuaOrbPattern

@Composable
fun LowPowerOrb() {
    // AUTO (default) follows Battery Saver; ON / OFF force it.
    // Low power = the spec's `performance.lowPower` block, else 30 fps with glow and particles off.
    SinuaOrb(pattern = SinuaOrbPattern.SPEAKING, lowPower = FxLowPower.AUTO, modifier = Modifier.size(160.dp))
}

File: apps/site/snippets/perf/low-power.swift
import SwiftUI
import Sinua

struct LowPowerOrb: View {
    var body: some View {
        // .auto (default) follows iOS Low Power Mode; .on / .off force it.
        // Low power = the spec's `performance.lowPower` block, else 30 fps with glow and particles off.
        SinuaOrb(pattern: .speaking, lowPower: .auto)
            .frame(width: 160, height: 160)
    }
}
